docker/api/build.py

```diff
@@ -42,6 +83,8 @@ def build(self, path=None, tag=None, quiet=False, fileobj=None,
             raise TypeError("You must specify a directory to build in path")
         else:
             dockerignore = os.path.join(path, '.dockerignore')
+            # If there are a lot of files, this takes time. Tell the user.
+            log.debug('taring and gzipping files: %s'%path)
```

Comment:
I don't think this is being gzipped.

The formatting is off a bit, there should be spaces before and after `%`.

I believe it's also common practice to use double quotes for strings that are messages to users (and single quotes for strings that are internal constants).